dozer-cli/src/ui_helper.rs

```diff
@@ -77,7 +80,7 @@ fn prepare_pipeline_dag(
             AppSourceMappings::new(connection.name.to_string(), ports_with_source_name),
         );
     });
-    statement_to_pipeline(&sql, &mut pipeline, None)?;
+    statement_to_pipeline(&sql, &mut pipeline, None, udfs.to_vec())?;
```

Comment:
We don't need this clone here. Let's think twice every time we write `.clone`, `.to_vec`, `.to_string` or `.to_owned`